TheaterSchedule.DAL/Repositories/SettingsRepository.cs

```diff
@@ -0,0 +1,60 @@
+﻿using Microsoft.EntityFrameworkCore;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Linq.Expressions;
+using System.Text;
+using TheaterSchedule.DAL.Contexts;
+using TheaterSchedule.DAL.Entities;
+using TheaterSchedule.DAL.Interfaces;
+
+namespace TheaterSchedule.DAL.Repositories
+{
+    class SettingsRepository : ISettingsRepository
+    {
+
+        private TheaterScheduleContext db;
+
+        public SettingsRepository(TheaterScheduleContext context)
+        {
+            this.db = context;
+        }
+
+        //Get
+        public Settings GetSettingsByPhoneId(string phoneId)
+        {
+           Settings settings = null;
+           settings  = db.Settings.Select(a => a).Where(a => a.Account.PhoneIdentifier== phoneId).SingleOrDefault();
+           return settings;
+        }
+
+        //Put
+        public void ChangeSettingsWithCurrentPhoneIdOrCreateNew(string phoneId, Settings settings)
+        {
+            Settings ResultSettings = null;
+            if (db.Settings.Any(s => s.Account.PhoneIdentifier == phoneId))
+            {            
+                ResultSettings = db.Settings.Select(a => a).Where(a => a.Account.PhoneIdentifier == phoneId).SingleOrDefault();
+                ResultSettings.LanguageId = settings.LanguageId;
```

Comment:
i think it will be better to accept language name (like 'ua', 'eng') to don't work with ids on frontend